pkg/cmd/miscellaneous.go

```diff
@@ -396,10 +398,23 @@ func getTargetName(Kind string) (string, error) {
 	return "", errors.New("Kind:" + Kind + "not found from ~/.garden/sessions/plantingSession/target")
 }
 
+//GetTargetName for isTargeted testing
+func GetTargetName(targetReader TargetReader, Kind string) (string, error) {
```

Comment:
It might be a good idea to give this a more explicit name from `getTargetName` - can we switch to `getTargetNameTest` if it's only being used for testing? Or `getTargetNameMisc`? 

Same for `isTargeted`/`IsTargeted`

Reply:
sure that makes sense to me, will switch to GetTargetNameTest and IsTargetedTest